### PR TITLE
fix: improve slider

### PR DESCRIPTION
**Pull request by natapg**
Closes [Web-218](https://fpjs.myjetbrains.com/youtrack/issue/WEB-218)

### CODE REVIEW

File: src/components/common/RangeSlider/RangeSlider.module.scss [hunk 1]
```diff
@@ -27,7 +27,7 @@
   }
 
   .label,
-  .onlyCurrentLabel {
+  .labelBelow {
```
Comment (natapg) [inline src/components/common/RangeSlider/RangeSlider.module.scss:30]: onlyCurrentLabel was an outdated name

File: src/components/common/RangeSlider/index.tsx [hunk 2]
```diff
@@ -9,28 +9,40 @@ export interface SliderValue {
...
   const handleSliderValueChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    ref.current?.setAttribute('step', 'any')
     const newValue = Number(e.target.value)
-    handleValueChange(newValue)
-    setSliderOffsetCss(calculateSliderOffset(min, max, newValue))
+    changeSliderValue(newValue)
+  }
+
+  const moveToClosestValue = () => {
+    ref.current?.setAttribute('step', '1')
```
Comment (natapg) [inline src/components/common/RangeSlider/index.tsx:43]: for firefox it's not enough to change the value of the slider, so I had to remove the step

**Comment by tan-90** [comment]
Have you checked it on Safari? It works for me on chrome but it seems to get stuck and jump around sometimes on Safari.

**Comment by natapg** [comment]
> Have you checked it on Safari? It works for me on chrome but it seems to get stuck and jump around sometimes on Safari.

please test again @tan-90

**Review by tan-90 — APPROVED** [review]

**Pull request merged**